sbot/py.typed

```diff
@@ -1 +0,0 @@
-# Marker file for PEP 561. sbot uses inline type hints. See PEP 484 for more information.
```

Comment:
Think we need this

Reply:
05a96aae1e73a3fe1a7ff92f3360334b24f3095e